src/PointwiseFunctions/Hydro/EquationsOfState/Tabulated3d.hpp

```diff
@@ -64,7 +64,7 @@ class Tabulated3D : public EquationOfState<IsRelativistic, 3> {
   using options = tmpl::list<FileName>;
 
   /// Fields stored in the table
-  enum { EPS = 0, PRESS, CS2, NUM_TAB3D_VARS };
+  enum { EPS = 0, PRESS, CS2, DELTA_MU, NUM_TAB3D_VARS };
```

Comment:
update `DELTA_MU` to `DeltaMu` once you've updated the previous PR and rebase this